modules/builtin/src/bot-templates/small-talk/flows/main.flow.json

```diff
@@ -9,9 +9,23 @@
     {
       "id": "entry",
       "name": "entry",
-      "next": [],
+      "next": [
+        {
+          "condition": "event.nlu.intent.name === 'none'",
+          "node": "none_message"
+        }
+      ],
       "onEnter": [],
       "onReceive": null
+    },
+    {
+      "id": "58a637911b",
+      "name": "none_message",
+      "next": [],
+      "onEnter": [
+        "say #!builtin_text-iVBzXn"
```

Comment:
There is no content elements in small talk, this will give an error

Reply:
sorry about that. will repush